libs/auth-api-lib/src/lib/delegations/delegations.service.ts

```diff
@@ -616,6 +743,32 @@ export class DelegationsService {
         )
         return d.toDTO()
       })
+
+    // Check live status, i.e. dead or alive for delegations
+    const {
+      aliveDelegations,
+      deceasedDelegations,
+    } = await this.getLiveStatusFromDelegations(delegationModels)
+
+    if (deceasedDelegations.length > 0) {
+      // Delete all deceased delegations by deleting them and their scopes.
+      const deletePromises = deceasedDelegations
+        .map(({ id }) =>
+          id ? this.delete(user, id, DelegationDirection.INCOMING) : undefined,
+        )
+        .filter(isDefined)
+
+      await Promise.all(deletePromises)
+
+      this.auditService.audit({
+        auth: user,
```

Comment:
Can you still pass both `auth` and `system: true`? 😮

Reply:
Ahh forgot never on property field, This does the trick
![image](https://user-images.githubusercontent.com/112904566/196137605-3e9421bf-51c7-4e67-911d-3d38cc4b44d8.png)